### PR TITLE
Update to call LDS instead of ALS for allocatedSeat number

### PR DESCRIPTION
**Pull request by thurstontye**
🐿 v2.5.16 Fixes incorrect seat allocation value on Overview and anywhere else that uses kmt-utilities.getLicenceSeatsInfo()

### CODE REVIEW

File: lib/middleware/getLicenceSeatsInfo.js [hunk 1]
```diff
@@ -13,23 +14,25 @@ module.exports = (req, res, next) => {
...
 
-		accessLicenceClient.getSeats(req.licenceId)
-			.then(currentSeats => {
-				const seatCount = Array.isArray(currentSeats) ? currentSeats.length : 0;
-				logger.info({operation, subOp: 'getSeats', licenceId: req.licenceId, allocatedSeatCount: seatCount});
-				req.licenceSeatsInfo.seatsAllocated = seatCount || 0;
+			licenceDataClient.getFilteredUserList(req.licenceId, req.apiAuthToken)
+			.then(licenceSeats => {
+				logger.info({operation, subOp: 'getSeats', licenceId: req.licenceId});
+				req.licenceSeatsInfo.seatsAllocated = Array.isArray(licenceSeats.seatHolders) ? licenceSeats.seatHolders.length : 0;
```
Comment (leggsimon) [inline lib/middleware/getLicenceSeatsInfo.js]: is that API call not paginated at all? I would've thought it would be

Comment (Thurst-ft) [reply lib/middleware/getLicenceSeatsInfo.js]: You're right, thougth if defaulted to all if no parma was set but it's actually 50 ☹️

Comment (leggsimon) [reply lib/middleware/getLicenceSeatsInfo.js]: So will it return a whole list of ~10,000 users if needs be?

Comment (Thurst-ft) [reply lib/middleware/getLicenceSeatsInfo.js]: Will now, upto 50,000 users. Was looking at calling getFilteredUserList recusively but the seatLimit value may not exist if the licences is effectively unlimited so not sure how we'd calculate the number of requests required. Our largest known licence is currentlly 13K. I could continue working on a solution but chosen to add this to the Kat tech debt given the best option for the future would be to refactor our api request to use the ft-core graphQL.

**Pull request merged**